docs/concepts/workloads/controllers/nginx-deployment.yaml

```diff
@@ -2,8 +2,13 @@ apiVersion: apps/v1beta1 # for versions before 1.6.0 use extensions/v1beta1
 kind: Deployment
 metadata:
   name: nginx-deployment
```

Comment:
API version should be changed to `apps/v1beta2`.

Reply:
Done.